### DISCUSSION TITLE
DST ends and epex acknowledge they will mess up the extra hour...

### DISCUSSION THREAD

**Discussion by deltamelter** [mampfes/ha_epex_spot#58] [General]
> We would like to inform you that we expect a display issue on our website, which may impact the Market Results display for all instruments on the Long Clock Change day. Therefore, we highly recommend referring to the Market Results as published on the Market Data sftp server

Doesn't seem like HA will cope with the extra hour anyway :)

**Comment by thomasdelaet** [comment]
what's the way to deal with this?

**Comment by mampfes** [comment]
If you use Awattar or SMARD.de, there is no problem. Only if you use the EPEX Spot Web Scraper, you will be off for 1 h because the EPEX Spot web page doesn't show values for the extra hour.
